src/components/Reports/Reports.js

```diff
@@ -98,13 +97,10 @@ class Reports extends React.Component {
                   withEmail
                 />
               </div>
-              {// TODO: Remove when component and datahub finish /
-              !this.state.disableTrafficSources ? (
-                <ReportsTrafficSources
-                  domainName={this.state.domainSelected.name}
-                  dateFrom={this.state.dateFrom}
-                />
-              ) : null}
+              <ReportsTrafficSources
+                domainName={this.state.domainSelected.name}
+                dateFrom={this.state.dateFrom}
+              />
```

Comment:
Is it already available? :)

Please, verify it well with a real production account before merging.